backend/controllers/formController.js

```diff
@@ -0,0 +1,148 @@
+const express =require ('express');
+const mongoose =require('mongoose');
+const Form= require('../models/FormModel');
+const Event= require('../models/EventModel');
+
+/*const createForm = async (req,res) => {
+    try{
+     const {eventId, price, data} = req.body;
+     const event = await Event.findBy(eventId);
+     if (!event) {
+        return res.status(400).json({error: "event does not exist"});
+     }
+      //create Form 
+      const newForm =new Form({
+        eventId,
+        price,
+        data
+    });
+    await newForm.save();
+    return newForm;
+ 
+    }catch(error){
+        console.log("Error in the login controller")
+        res.status(500).json({error:"Internal Server Error"})
+     }
+    } */
+// Create a new Form
+ const createForm = async (eventId,description, price, data) => {
+    try{
+        // Check if the event exists
+        const event = await Event.findById(eventId);
+        if (!event) {
+            throw new Error('Event not found');
```

Comment:
Don't through the error. Handle it
```suggestion
            res.status(400).json({message:"No event found"})
```